src/sentry/static/sentry/app/utils/projects.tsx

```diff
@@ -138,9 +227,9 @@ class Projects extends React.Component {
     // For each item in the fetch queue, lookup the project object and in the case
     // where something wrong has happened and we were unable to get project summary from
     // the server, just fill in with an object with only the slug
-    const projectsOrPlaceholder = Array.from(this.fetchQueue).map(slug =>
-      projectsMap.has(slug) ? projectsMap.get(slug) : {slug}
```

Comment:
Typescript does not understand this ternary since `map.get()` can potentially be undefined.